libs/editor/egui_editor/src/input/mutation.rs

```diff
@@ -237,6 +235,9 @@ pub fn calc(
                     } else {
                         continue; // only process list items
                     };
+                if !indentation_processed_galleys.insert(galley_idx) {
+                    continue; // only process each galley once
+                }
```

Comment:
first, because this was happening before other skips, any galley intersecting in the cursor ended up in this set...